03_launch_mgmt_cluster.sh

```diff
@@ -90,31 +90,45 @@ function update_kustomization_images(){
   for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
     IMAGE=${!IMAGE_VAR}
     #shellcheck disable=SC2086
-    IMAGE_NAME="${IMAGE##*/}:latest"
+    IMAGE_NAME="${IMAGE##*/}"
     LOCAL_IMAGE="${REGISTRY}/localimages/$IMAGE_NAME"
-
     OLD_IMAGE_VAR="${IMAGE_VAR%_LOCAL_IMAGE}_IMAGE"
     # Strip the tag for image replacement
     OLD_IMAGE="${!OLD_IMAGE_VAR%:*}"
     #shellcheck disable=SC2086
     kustomize edit set image $OLD_IMAGE=$LOCAL_IMAGE
   done
+  for IMAGE_VAR in $(env | grep -v "_LOCAL_IMAGE=" | grep "_IMAGE=" | grep -o "^[^=]*") ; do
+    IMAGE=${!IMAGE_VAR}
+    #shellcheck disable=SC2086
+    IMAGE_NAME="${IMAGE##*/}"
+    LOCAL_IMAGE="${REGISTRY}/localimages/$IMAGE_NAME"
+    #shellcheck disable=SC2086
+    kustomize edit set image $IMAGE=$LOCAL_IMAGE
+  done
 }
 
 # Modifies the images to use the ones built locally
 function update_images(){
   for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
     IMAGE=${!IMAGE_VAR}
     #shellcheck disable=SC2086
-    IMAGE_NAME="${IMAGE##*/}:latest"
+    IMAGE_NAME="${IMAGE##*/}"
     LOCAL_IMAGE="${REGISTRY}/localimages/$IMAGE_NAME"
-
     OLD_IMAGE_VAR="${IMAGE_VAR%_LOCAL_IMAGE}_IMAGE"
     # Strip the tag for image replacement
     OLD_IMAGE="${!OLD_IMAGE_VAR%:*}"
     eval "$OLD_IMAGE_VAR"="$LOCAL_IMAGE"
     export "${OLD_IMAGE_VAR?}"
   done
+  for IMAGE_VAR in $(env | grep -v "_LOCAL_IMAGE=" | grep "_IMAGE=" | grep -o "^[^=]*") ; do
+    IMAGE=${!IMAGE_VAR}
+    #shellcheck disable=SC2086
+    IMAGE_NAME="${IMAGE##*/}"
+    LOCAL_IMAGE="${REGISTRY}/localimages/$IMAGE_NAME"
+    # Strip the tag for image replacement
+    eval "$IMAGE_VAR"="$LOCAL_IMAGE"
```

Comment:
This will not be applied in the patch_clusterctl function. You need to find a way to use it there.

Reply:
Update done. thanks